src/api.js

```diff
@@ -376,3 +376,61 @@ api.declare({
   }
   return res.reply({installed: false});
 });
+
+// Below is the draft code to make the badges clickable.
+// In the finished thing, the helper function will go to the beginning of the module,
+// and will be used in other APIs as well.
+api.declare({
+  name: 'taskLink',
+  title: 'Redirects to the task inspector page',
+  description: [
+    'Builds a link to the task inspector for the given task group',
+    ' and redirects the user to that page.',
+  ].join('\n'),
+  stability: 'experimental',
+  method: 'get',
+  route: '/taskLink/:owner/:repo/:branch',
+}, async function(req, res) {
+  // Extract owner, repo and branch from request into variables
+  let {owner, repo, branch} = req.params;
+
+  let authenticate = installationAuthenticate.bind(this);
+  let instGithub = await authenticate(owner);
+
+  // Get task group ID
+  if (instGithub) {
+    try {
+      let findStatus = findTCstatus.bind(this);
```

Comment:
Rather than bind'ing to a function, I'd prefer to just pass the context explicitly:

```
await findStatus(instGithub, owner, repo, branch, this)
```
...
```
async function findTCstatus(github, owner, repo, branch, context) {
... context.cfg.app.botName ...
}
```
(same for `installationAuthenticate`, above)